client/src/Components/About/About.js

```diff
@@ -114,13 +125,14 @@ const About = () => {
           formatSection(section, index % 2 !== 0)
         )}
         {!user && (
-          <Grid item className={style.center} xs={12}>
+          <Grid item className={style.footer} xs={12}>
```

Comment:
You can leave the `className={style.center}` here and add a `style={{ marginTop: '8px' }}` property to accomplish the same. Please note that we use to use multiples of 8 for spacing.